lib/generators/solid_cache/install/templates/create_solid_cache_entries_hash.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSolidCacheEntries < ActiveRecord::Migration[7.0]
+    def change
+      create_table :solid_cache_entries do |t|
+        t.binary   :key,        null: false,   limit: 1024
+        t.binary   :value,      null: false,   limit: 512.megabytes
+        t.datetime :created_at, null: false
+
+        t.index    :key,        unique: true,  using: :hash
```

Comment:
Error encountered with message
`PG::FeatureNotSupported: ERROR:  access method "hash" does not support unique indexes`

From doc
https://www.postgresql.org/docs/16/indexes-unique.html
`Currently, only B-tree indexes can be declared unique.`

Reply:
Though I attempted to workaround it by removing `unique: true`, I would encounter another error
`ArgumentError - No unique index found for key` from
https://github.com/rails/rails/blob/v7.1.2/activerecord/lib/active_record/insert_all.rb#L161